index.html

```diff
@@ -2,10 +2,11 @@
 <html lang="en">
     <head>
         <meta charset="UTF-8" />
-        <link rel="icon" href="/favicon.ico" />
+        <link rel="icon" type="image/svg+xml" href="/favicon/favicon.svg">
+        <link rel="icon" type="image/png" href="/favicon/favicon.png">
         <!-- Disallow zooming on touch, as the zoom gesture is already reserved for zooming the map -->
         <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
-        <title>Vite App</title>
+        <title>Maps of Switzerland - Swiss Confederation - map.geo.admin.ch</title>
```

Comment:
Should not be this localized based on the selected language ?

Reply:
It is later on in `App.vue`, I just wanted to avoid having "dev" placeholder further down the line (if for instance something goes wrong with JS and nothing is loaded)